packages/aws-cdk-lib/aws-ec2/lib/machine-image/machine-image.ts

```diff
@@ -728,5 +746,11 @@ export interface LookupMachineImageProps {
    * @default - Empty user data appropriate for the platform type
    */
   readonly userData?: UserData;
+
+  /**
+   * When the image is cached in cdk.context.json, adds an additional discriminator to the
+   * cache key so that separate lookups with the same parameters can have separate cache lifecycles
```

Comment:
For clarity I'm thinking "If cachedInContext is true, adds an additional discriminator..." (since this variable does nothing on its own)

Reply:
Hm, unfortunately cachedInContext does not always appear on the lookup parameters object. Probably better not to do it for the sake of consistency then. If anything it would be "adds an additional discriminator... if cached", but not sure how helpful that really is at that point